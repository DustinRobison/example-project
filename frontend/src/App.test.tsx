import React from "react";
import { render } from "@testing-library/react";
import { act } from "react-dom/test-utils";
import App from "./App";
import { MockedProvider } from "@apollo/client/testing";

test("Render Top Level App", async () => {
  let AppElement;

  act(() => {
    AppElement = render(
      <MockedProvider mocks={[]}>
        <App />
      </MockedProvider>
    );
  });

  expect(AppElement).toMatchSnapshot();
});
