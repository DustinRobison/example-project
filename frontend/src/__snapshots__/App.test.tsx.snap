// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Top Level App 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="makeStyles-content-1"
      >
        <div
          class="MuiPaper-root makeStyles-root-2 MuiPaper-elevation1 MuiPaper-rounded"
        >
          <h4
            class="MuiTypography-root MuiTypography-h4 MuiTypography-alignCenter"
          >
            Flowers
          </h4>
          <div
            class="makeStyles-content-3"
          >
            <div
              class="MuiTabs-root"
            >
              <div
                class="MuiTabs-scroller MuiTabs-fixed"
                style="overflow: hidden;"
              >
                <div
                  class="MuiTabs-flexContainer MuiTabs-centered"
                  role="tablist"
                >
                  <button
                    aria-selected="true"
                    class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary Mui-selected"
                    role="tab"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiTab-wrapper"
                    >
                      Flowers
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                  <button
                    aria-selected="false"
                    class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary"
                    role="tab"
                    tabindex="-1"
                    type="button"
                  >
                    <span
                      class="MuiTab-wrapper"
                    >
                      Family
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </div>
                <span
                  class="PrivateTabIndicator-root-4 PrivateTabIndicator-colorPrimary-5 MuiTabs-indicator"
                  style="left: 0px; width: 0px;"
                />
              </div>
            </div>
            <div>
              <div
                class="MuiPaper-root makeStyles-root-8 MuiPaper-outlined MuiPaper-rounded"
              >
                <ul
                  class="MuiList-root MuiList-padding"
                >
                  Loading...
                  <li
                    class="MuiListItem-root MuiListItem-gutters"
                  >
                    <button
                      class="MuiButtonBase-root MuiButton-root MuiButton-contained"
                      tabindex="0"
                      type="button"
                    >
                      <span
                        class="MuiButton-label"
                      >
                        Add Flower
                      </span>
                      <span
                        class="MuiTouchRipple-root"
                      />
                    </button>
                  </li>
                </ul>
              </div>
            </div>
            <div
              hidden=""
            >
              <div
                class="MuiPaper-root makeStyles-root-10 MuiPaper-outlined MuiPaper-rounded"
              >
                <ul
                  class="MuiList-root MuiList-padding"
                >
                  Loading...
                </ul>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="makeStyles-content-1"
    >
      <div
        class="MuiPaper-root makeStyles-root-2 MuiPaper-elevation1 MuiPaper-rounded"
      >
        <h4
          class="MuiTypography-root MuiTypography-h4 MuiTypography-alignCenter"
        >
          Flowers
        </h4>
        <div
          class="makeStyles-content-3"
        >
          <div
            class="MuiTabs-root"
          >
            <div
              class="MuiTabs-scroller MuiTabs-fixed"
              style="overflow: hidden;"
            >
              <div
                class="MuiTabs-flexContainer MuiTabs-centered"
                role="tablist"
              >
                <button
                  aria-selected="true"
                  class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary Mui-selected"
                  role="tab"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiTab-wrapper"
                  >
                    Flowers
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
                <button
                  aria-selected="false"
                  class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary"
                  role="tab"
                  tabindex="-1"
                  type="button"
                >
                  <span
                    class="MuiTab-wrapper"
                  >
                    Family
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </div>
              <span
                class="PrivateTabIndicator-root-4 PrivateTabIndicator-colorPrimary-5 MuiTabs-indicator"
                style="left: 0px; width: 0px;"
              />
            </div>
          </div>
          <div>
            <div
              class="MuiPaper-root makeStyles-root-8 MuiPaper-outlined MuiPaper-rounded"
            >
              <ul
                class="MuiList-root MuiList-padding"
              >
                Loading...
                <li
                  class="MuiListItem-root MuiListItem-gutters"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-contained"
                    tabindex="0"
                    type="button"
                  >
                    <span
                      class="MuiButton-label"
                    >
                      Add Flower
                    </span>
                    <span
                      class="MuiTouchRipple-root"
                    />
                  </button>
                </li>
              </ul>
            </div>
          </div>
          <div
            hidden=""
          >
            <div
              class="MuiPaper-root makeStyles-root-10 MuiPaper-outlined MuiPaper-rounded"
            >
              <ul
                class="MuiList-root MuiList-padding"
              >
                Loading...
              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
